Sources/Testing/ABI/EntryPoints/EntryPoint.swift

```diff
@@ -295,6 +298,7 @@ extension __CommandLineArguments_v0: Codable {
     case skip
     case repetitions
     case repeatUntil
+    case experimentalAttachmentPath
```

Comment:
Should its raw value be `"_experimentalAttachmentPath`" or some other underscore-prefixed string?